src/Microsoft.Diagnostics.EventFlow.Core/Implementations/EventFlowJsonUtilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Diagnostics.EventFlow
+{
+    public class EventFlowJsonUtilities
```

Comment:
Make this static class? Is it possible to make it internal if not used externally?

Reply:
This has to be a public class because code from other EventFlow assemblies is using it.

I made it static